app/controllers/genres_controller.rb

```diff
@@ -0,0 +1,49 @@
+class GenresController < ApplicationController
+  layout "genres"
+  before_action :find_genre, only: [:edit, :update, :destroy, :toggle_featured]
```

Comment:
There is an action here that doesn't belong. Can you find it and remove it?

Reply:
Oh my, it's `:toggle_featured`. Removing in the next commit!